safe/impact_function/test/test_impact_function.py

```diff
@@ -415,6 +414,10 @@ def test_minimum_extent(self):
             message
         )
 
+    @unittest.skipIf(
+        os.environ.get('QGIS_VERSION_TAG') == 'release-3_4',
```

Comment:
I saw your trick to add the docker image as environment variable. You could use the QGIS API for that ;-) Like https://github.com/inasafe/inasafe/blob/develop/safe/utilities/gis.py#L366

Reply:
Thanks for the hint!
That is a better way.
I asked @ann26 to continue with the PR, but she submit it to my repo instead.
I will merge it after this.
https://github.com/lucernae/inasafe/pull/4

Reply:
done, I've merged in here @Gustry @lucernae